ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
Please add a warning in this file that this is best effort and not maintained via CI.